.actions/assistant.py

```diff
@@ -163,6 +164,27 @@ def mirror_pkg2source(pypi_folder: str, src_folder: str) -> None:
                 shutil.rmtree(py_dir2, ignore_errors=True)
                 shutil.copytree(py_dir, py_dir2)
 
+    @staticmethod
+    def copy_replace_imports(
+        source_dir: str, source_import: str, target_import: str, target_dir: Optional[str] = None
```

Comment:
Can we have a test for this function, please? I'm not 100% sure it covers all possible edge cases (i.e. recently introduced graveyard)

Reply:
Agree, so run a test on our codebase?

Reply:
I meant a pytest function, so that we can have it tested all the time in CI so that we know we didn't introduce any new pattern which would break this

Reply:
I see... We shall add a test for most of the Assistant methods... :innocent: